peps/pep-0744.rst

```diff
@@ -501,9 +509,22 @@ Currently, the JIT is `about as fast as the existing specializing interpreter
 <https://github.com/faster-cpython/benchmarking-public/blob/main/configs.png>`__
 on most platforms. Improving this is obviously a top priority at this point,
 since providing a significant performance gain is the entire motivation for
-having a JIT at all. A number of proposed improvements are already underway, and
-this ongoing work is being tracked in `GH-115802
-<https://github.com/python/cpython/issues/115802>`__.
+having a JIT at all.
+
+Presently, there are a both higher-level and lower-level optimizations being 
```

Comment:
On further reflection, I'm actually not sure we need this paragraph, since it's sort of getting a bit too low-level. I also don't want to have a listing here that needs to be updated over time (and a lot of these are actually independent of the specific machine-code generation strategy proposed here).